internal/corazawaf/rule.go

```diff
@@ -167,9 +167,7 @@ func (r *Rule) Evaluate(phase types.RulePhase, tx plugintypes.TransactionState,
 }
 
 func (r *Rule) doEvaluate(phase types.RulePhase, tx *Transaction, cache map[transformationKey]*transformationValue) []types.MatchData {
-	if r.Capture {
-		tx.Capture = true
-	}
+	tx.Capture = r.Capture
 	rid := r.ID_
 	if rid == 0 {
```

Comment:
Again, constant use with semantic meaning instead of `0`?

Reply:
Something like `noID` would work I think.